### PR TITLE
Rename caption to label variant; move help text token to editable

### PR DESCRIPTION
**Pull request by libertymayc**
Merge caption and help text tokens to label variant
Update line heights

### CODE REVIEW

**Review by joshwooding — APPROVED** [review]

**Review by ZarkoPernar — DISMISSED** [review]

**Review by alycrys — APPROVED** [review]

**Review by joshwooding — CHANGES_REQUESTED** [review]
Some recent conversations have affected this change @pseys could you advise

**Comment by libertymayc** [comment]
> Some recent conversations have affected this change @pseys could you advise

done @joshwooding

**Review by joshwooding — APPROVED** [review]

**Review by alycrys — DISMISSED** [review]

File: packages/core/src/card/Card.css [hunk 1]
```diff
@@ -30,7 +30,7 @@
 a:focus .uitkCard-interactable,
 .uitkCard-interactable:hover {
   border-color: var(--uitkCard-borderColor-hover, var(--uitk-selectable-borderColor-hover));
-  color: var(--uitkCard-color-hover, var(--uitk-text-primary-foreground));
+  color: var(--uitkCard-color-disabled, var(--uitk-text-primary-foreground-hover));
```
Comment (joshwooding) [inline packages/core/src/card/Card.css:33]: Should this be disabled?

File: packages/theme/css/characteristics/text.css [hunk 2]
```diff
@@ -59,8 +55,13 @@
   --uitk-text-background-selected: var(--uitk-palette-neutral-highlight);
 
   --uitk-text-primary-foreground: var(--uitk-palette-neutral-primary-foreground);
+  --uitk-text-primary-foreground-hover: var(--uitk-palette-neutral-primary-foreground);
+  --uitk-text-primary-foreground-selected: var(--uitk-palette-neutral-primary-foreground);
   --uitk-text-primary-foreground-disabled: var(--uitk-palette-neutral-primary-foreground-disabled);
+
   --uitk-text-secondary-foreground: var(--uitk-palette-neutral-secondary-foreground);
+  --uitk-text-secondary-foreground-hover: var(--uitk-palette-neutral-secondary-foreground);
+  --uitk-text-secondary-foreground-selected: var(--uitk-palette-neutral-secondary-foreground);
```
Comment (joshwooding) [inline packages/theme/css/characteristics/text.css:64]: These aren't mentioned in the changeset

Comment (joshwooding) [reply packages/theme/css/characteristics/text.css:64]: Also do they need to be added to the doc page

File: packages/core/src/card/Card.css (hunk 1 above)
Comment (libertymayc) [reply packages/core/src/card/Card.css:33]: oh yes i fixed that in another one and then this has the old in still, will delete

File: packages/theme/css/characteristics/text.css (hunk 2 above)
Comment (libertymayc) [reply packages/theme/css/characteristics/text.css:64]: thanks, these also were deleted previously got added back in merge, shouldnt be here!

**Review by joshwooding — APPROVED** [review]

**Pull request merged**